README.md

```diff
@@ -856,6 +856,17 @@ cd tcl
 tclsh ./stepX_YYY.tcl
 ```
 
+### TypeScript
```

Comment:
Please add a credit line to yourself! (see example in the TCL implementation just above).